WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -286,13 +287,21 @@ private extension ProductsViewController {
 
         let productSKUBarcodeScannerCoordinator = ProductSKUBarcodeScannerCoordinator(sourceNavigationController: navigationController,
                                                                                       onSKUBarcodeScanned: { [weak self] scannedBarcode in
+            guard let self = self else { return }
```

Comment:
```suggestion
            guard let self else { return }
```
Just small, but no need for the `= self` any more

Reply:
Good suggestion! Done in 6eef8adf0444577088efe5e17bf9c180da2d4235